src/enzo/global_data.h

```diff
@@ -967,17 +967,27 @@ EXTERN char LocalPath[MAX_LINE_LENGTH];
 EXTERN char GlobalPath[MAX_LINE_LENGTH];
 #endif
 
-#ifdef USE_PYTHON
 EXTERN int NumberOfPythonCalls;
 EXTERN int NumberOfPythonTopGridCalls;
 EXTERN int NumberOfPythonSubcycleCalls;
+#ifdef USE_PYTHON
```

Comment:
Why was this `#ifdef` moved? The `USE_PYTHON` and `USE_LIBYT` compilation options should enable completely separate machinery.

Reply:
I don't remember why I did this, but if I had to hazard a guess it was because I was attempting to isolate the *actual* Python objects.